inlong-dashboard/src/ui/components/FieldList/index.tsx

```diff
@@ -30,17 +30,18 @@ export interface Props {
   inlongStreamId?: string;
   isSource: boolean;
   columns: ColumnsType;
-  //   readonly?: string;
 }
 
 const Comp: React.FC<Props> = ({ inlongGroupId, inlongStreamId, isSource, columns }) => {
   const [createModal, setCreateModal] = useState<Record<string, unknown>>({
     open: false,
   });
 
+  const [blur, setBlur] = useState(false);
```

Comment:
I think the newly added `blur` attribute is unnecessary. Can the existing props be used to complete the function?

On the other hand, passing `blur` to Modal feels meaningless.